source/numerics.tex

```diff
@@ -2143,7 +2143,7 @@
 \end{itemdescr}
 
 \begin{itemdecl}
-void seed();  
+void A::seed();
```

Comment:
I don't think `tuple`, `pair` and `vector` are really comparable. They are standard types defined with a synopsis and a detailed description of each function like a lot of others.

This section lists "adapter requirements" that any type can meet, and is more similar to the iterator categories, Allocator requirements, etc., which we specify by listing expressions in a table that must be valid and have some semantics. In fact the surrounding subclauses [rand.req.seedseq], [rand.req.urng], [rand.req.eng] and  [rand.req.dist] all use this approach too. I have no idea why [rand.req.adapt] is like this. For now I'll leave it alone, just removing the `void`.